tests/baselines/reference/callbackTagNamespace.types

```diff
@@ -16,7 +16,7 @@ var x = 1;
 /** @type {NS.Nested.Inner} */
 function f(space, peace) {
 >f : (space: any, peace: any) => string | number
->  : ^     ^^   ^^     ^^   ^^^^^^^^^^^^^^^^^^^^
+>  : ^     ^^^^^^^     ^^^^^^^^^^^^^^^^^^^^^^^^^

```

Comment:
How does reuse end up being lowered here?

Reply:
The original version for JsDoc did fall back to type printing, but it also set the original node on this synthetic type node counting it as a reuse. Falling back to type printing should not count the node as reused in my opinion. 

```ts
if (isTypeReferenceNode(node) && isInJSDoc(node) && (!existingTypeNodeIsNotReferenceOrIsReferenceWithCompatibleTypeArgumentCount(node, getTypeFromTypeNode(node)) || getIntendedTypeFromJSDocTypeReference(node) || unknownSymbol === resolveTypeReferenceName(node, SymbolFlags.Type, /*ignoreErrors*/ true))) {
                    return setOriginalNode(typeToTypeNodeHelper(getTypeFromTypeNode(node), context), node);
                }
                
```